src/main/java/com/farion/onlinebookstore/dto/book/BookDtoWithoutCategoryIds.java

```diff
@@ -0,0 +1,15 @@
+package com.farion.onlinebookstore.dto.book;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDtoWithoutCategoryIds {
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImag;
```

Comment:
```suggestion
    private String coverImage;
```
or is there a good reason for writing not a full word?